addons/l10n_ec_ats/account_invoice/views/factura_report.xml

```diff
@@ -0,0 +1,176 @@
+                        <small> <br></br>
+                        <div class="col-xs-8 col-xs-offset-2">
+                            <p class="text-center" t-field="o.payment_term_id"/>
+                        </div>
+                        <div class="col-xs-2">
+                            <p class="text-right" t-field="o.date_invoice" t-field-options='{"format":"d MMMM y"}'/>
+                        </div> </small>
+                    </div>
+                    <br></br>
+                    <div class="row" style="font-size:10px">
+                        <small> <div class="col-xs-2">
+                            <p class="text-left" t-field="o.name"/>
+                        </div>
+                        <div class="col-xs-2 col-xs-offset-8">
+                            <p class="text-right" t-field="o.date_due" t-field-options='{"format":"d MMMM y"}'/>
+                        </div> </small>
+                    </div>
+                    <br></br> <br></br>
+
+                    <div class="row" t-foreach="o.invoice_line_ids" t-as="i" style="font-size:10px">
+                        <small> <div class="col-xs-2">
+                            <span t-field="i.product_id.default_code"/>
+                        </div>
+                        <div class="col-xs-6">
+                            <span t-field="i.product_id.name"/> <![CDATA[&nbsp;]]> <span t-field="i.product_id.barcode"/>
+                        </div>
+                        <div class="col-xs-1" style="text-align:right">
+                            <span t-field="i.quantity"/>
+                        </div>
+                        <div class="col-xs-1" style="text-align:right">
+                            <span t-field="i.price_unit"/>
+                        </div>
+                        <div class="col-xs-2" style="text-align:right">
+                            <span t-field="i.price_subtotal"
+                                          t-field-options='{"widget": "monetary", "display_currency": "o.currency_id"}'/>
+                        </div> </small>
+                    </div>
+
+                    <div class="row">
+
+                        <div class="col-xs-7 col-xs-offset-5" style="font-size:10px">
+                            <!-- <address t-field="o.partner_id"
+                            t-field-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}' /> -->
+                            <div class="col-xs-8 col-xs-offset-2">
+                                ESTE ES UN EJEMPLO DE FACTURA<br></br> SI DESEA UNA PERSONALIZACION GRATIS <br></br> ESCRIBA UN EMAIL A: <br></br>
+                                snowblow07@gmail.com <br></br> Y PONGA EN EL TITULO "YO APOYO EL SOFTWARE LIBRE"
+                        </div>
+                            <!-- <span t-if="o.partner_id.vat">TIN: <span t-field="o.partner_id.vat"/></span> -->
+                        </div>
+                    </div>
+
+

```

Comment:
This should be on the demo folder and must be installed only if the user want the demo data. It should not be on the base module as some people would not need or want this, and it will conflict.